plugins/tutor-contrib-paragon/tutorparagon/plugin.py

```diff
@@ -34,6 +34,11 @@
         # Paragon Builder Docker image
         # This image is used to compile themes and should be built with `tutor images build paragon-builder`
         ("PARAGON_BUILDER_IMAGE", "paragon-builder:latest"),
+        # Paragon static server configuration
+        # This server serves the compiled themes
+        ("PARAGON_STATIC_SERVER_IMAGE", "caddy:alpine"),
+        ("PARAGON_STATIC_SERVER_PORT", 18000),
```

Comment:
I'm curious about this port choice. It seems like a perfectly reasonable one, I'd just like to make sure it won't conflict with other plugins/services.